packages/core/package.json

```diff
@@ -50,6 +50,8 @@
   "dependencies": {
     "@popperjs/core": "^2.11.8",
     "@stencil/core": "^4.9.0",
+    "date-fns": "^2.30.0",
+    "dotenv": "^16.0.3",
```

Comment:
We dropped "dotenv" a couple of releases ago due to the new Node version supporting it, do we need it for something here?

Reply:
Nope, not needed. I'll update